terraform/README.md

```diff
@@ -199,46 +199,45 @@ Lastly, the script creates a Cloud Scheduler running once per day to trigger Clo
 |---|---|:---:|:---:|:---:|
 | [project_id](variables.tf#L17) | Identifier of an existing GCP project for GKE Policy Automation resources. | `string` | ✓ |  |
 | [region](variables.tf#L22) | GCP region for GKE Policy Automation resources. | `string` | ✓ |  |
+| [discovery](variables.tf#L51) | Configuration of cluster discovery mechanism. Check [discovery attributes](#discovery-attributes).  | `object` | ✓ |  |
 | [job_name](variables.tf#L27) | Name of a Cloud Run Job for GKE Policy Automation container. | `string` |  | `gke-policy-automation` |
-| [config_file_path](variables.tf#L33) | Path to the YAML file with GKE Policy Automation configuration. | `string` |  | `config.yaml` |
-| [config_file_path](variables.tf#L39) | CRON interval for triggering the GKE Policy Automation job. | `string` |  | `"0 1 * * *` |
+| [config_file_path](variables.tf#L33) | Path to the YAML file with [GKE Policy Automation configuration](../docs/user-guide.md#configuration-file). | `string` |  | `config.yaml` |
+| [cron_interval](variables.tf#L39) | CRON interval for triggering the GKE Policy Automation job. | `string` |  | `"0 1 * * *` |
 | [run_script](variables.tf#L45) | Indicates whether to run script for populating Artifact Registry and Cloud Run Jobs. | `bool` |  | `false` |
-| [discovery](variables.tf#L51) | Configuration of cluster discovery mechanism. Check [discovery attributes](#discovery-attributes).  | `map(any)` |  | `{"enabled" = true}` |
-| [output_storage](variables.tf#L63) | Configuration of Cloud Storage output. Check [Cloud Storage attributes](#cloud-storage-attributes). | `map(any)` |  | `{"enabled" = false}` |
-| [output_pubsub](variables.tf#L45) | Configuration of Pub/Sub output. Check [Pub/Sub attributes](#pubsub-attributes) | `map(any)` |  | `{"enabled" = false}` |
-| [output_scc](variables.tf#L99) | Configuration of Security Command Center output. Check [Security Command Center attributes](#security-command-center-attributes). | `map(any)` |  | `{"enabled" = false}` |
+| [output_storage](variables.tf#L64) | Configuration of Cloud Storage output. Check [Cloud Storage attributes](#cloud-storage-attributes). | `object` |  | `{"enabled" = false}` |
+| [output_pubsub](variables.tf#L84) | Configuration of Pub/Sub output. Check [Pub/Sub attributes](#pubsub-attributes) | `object` |  | `{"enabled" = false}` |
+| [output_scc](variables.tf#L99) | Configuration of Security Command Center output. Check [Security Command Center attributes](#security-command-center-attributes). | `object` |  | `{"enabled" = false}` |
 
 ### Discovery attributes
 
 | Name | Description | Type | Required | Default |
 |---|---|:---:|:---:|:---:|
-| [enabled](variables.tf#L51) | Indicates if resources for discovery mechanism will be provisioned. | `bool` | ✓ |  |
```

Comment:
Good point about removing.
In general I assumed that server-less without discovery makes limited sense.
The `discovery` object I use to provision both, asset inventory and container API IAM roles. In case we want to have server-less without discovery, then introduction of another object (to provision GKE API access) will be needed.